### PR TITLE
Removed celery, refactor hardware control

### PR DESCRIPTION
**Pull request by 0phelia36**
## TL;DR
Removes celery, the program now runs as two separate processes, one for flask, another for controlling the hardware, communicating through a queue.

Fixes the issue where two different versions of the hardware code were running at the same time in celery and in flask. 

Makes hardware control and configuration outside of recipes simpler.

Tasks are now run periodically by a thread on the hardware process, rather than continuously by a celery task.

## What
What is affected by this PR?
- [ ] API
- [ ] GUI
- [X] Hardware Integration
- [X] OS/Deployment
- [ ] Documentation
- [ ] Other (please describe in notes)

## Testing
How was this tested?
- [ ] Locally Virtualized
- [ X ] Raspberry Pi
- [ ] With Hardware
- [ ] Other (please describe in notes)

## Notes
How was your day?

### CODE REVIEW

File: backend/recipes/base.py [hunk 1]
```diff
@@ -293,15 +288,41 @@ def areTasksComplete(self):
...
 
+    def tickTasks(self):
+        """
+        Executes one iteration of the current tasks that are scheduled to run.
+        :return:
+            None
+        """
+        for task in self.currentTasks:
+            if not task["done"] and datetime.now() > task["nextTime"]:
+                print("task is ready for next iteration")
```
Comment (sparklestheunicorn) [inline backend/recipes/base.py:311]: Can we remove this (and other fluffy print statements that would appear a lot in the logs)? Errors and whatnot are great but I've filled up a lot of disk space accidentally when I left my testing unit on for an extended period with some extraneous debugging print.

**Comment by sparklestheunicorn** [comment]
This looks good to me. I'll try it out on the full rig when I get my dev unit set back up again.
